env/README.md

```diff
@@ -0,0 +1,46 @@
+# Setting up environments for The Data Science Community of Practice scipts/notebooks
+
+The environments for code found in this repository, either in scripts or in notebooks, depending on the platform and type of work you are doing, are all primarily provided by conda environments.
+
+To create an environment from any of the *.yml* files, run the following command
+```bash
+conda env create --file requirement.yml
+```
+where you replace `requirements.yml` with the particular requirements file in the folder required for the files you want to run. There several different environments as different packages are required for different activities or demonstrations, some of which are mutually incompatible or make for slow/difficult to resolve installations, so we have tried to keep each environment as simple as possible to improve performance, as well as fixing compatible major and minor package versions to help environment solving.
+
```

Comment:
It would be good to say something about. how these are general libraries which are intended as starting points for creating other environments for projects with more niche requirements. For example, 
```
These files provide  pre-checked base environments that are a starting point for project environments into which other more project specific  libraries can be added.
```